tests/sentry/utils/performance_issues/test_consecutive_http_detector.py

```diff
@@ -97,12 +123,101 @@ def test_does_not_detect_consecutive_http_issue_with_frontend_events(self):
         problems = self.find_problems(event)
         assert problems == []
 
-    def test_does_not_detect_consecutive_http_issue_with_low_duration(self):
-        event = self.create_issue_event(100)
-        problems = self.find_problems(event)
+    def test_does_not_detect_consecutive_http_issue_with_low_count(self):
+        spans = [  # count less than threshold
+            create_span("http.client", 20, "GET /api/0/organizations/endpoint1", "hash1"),
+        ]
+
+        problems = self.find_problems(create_event(spans))
+        assert problems == []
+
+    def test_detects_consecutive_http_issue_with_trailing_low_duration_span(self):
+        spans = [
+            create_span(
+                "http.client", 900, "GET /api/0/organizations/endpoint1", "hash1"
```

Comment:
You may want to extract the `900` into a constant and only change that as you're decreasing the thresholds, it'll just make changing a lot of these test parameters easier in the future PRs

Reply:
Yeah sounds good.